virttest/utils_misc.py

```diff
@@ -2298,7 +2298,7 @@ def get_qemu_best_cpu_model(params):
     return params.get("default_cpu_model", None)
 
 
-def get_qemu_version(params):
+def get_qemu_version(params={}):
```

Comment:
Please do not use mutable objects as function params, please refer to https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html.

Reply:
thx for the tips, updated